build/charts/antrea/templates/_helpers.tpl

```diff
@@ -9,17 +9,21 @@
 {{- end -}}
 
 {{- define "antreaImageTag" -}}
-{{- if .Values.image.tag }}
-{{- .Values.image.tag -}}
+{{- if .Values.agentImage.tag }}
+{{- .Values.agentImage.tag -}}
 {{- else if eq .Chart.AppVersion "latest" }}
 {{- print "latest" -}}
 {{- else }}
 {{- print "v" .Chart.AppVersion -}}
 {{- end }}
 {{- end -}}
 
-{{- define "antreaImage" -}}
-{{- print .Values.image.repository ":" (include "antreaImageTag" .) -}}
+{{- define "antreaControllerImage" -}}
+{{- print .Values.controllerImage.repository ":" (include "antreaImageTag" .) -}}
+{{- end -}}
+
+{{- define "antreaAgentImage" -}}
+{{- print .Values.agentImage.repository ":" (include "antreaImageTag" .) -}}
```

Comment:
the way the logic should work is:
- if `.Values.image` is defined, it should override `.Values.agentImage` and `.Values.controllerImage`
- by default, `.Values.image` should be unset / undefined